apps/client/src/components/Markdown/renderers/ListRenderer.js

```diff
@@ -1,4 +1,4 @@
 import React from "react";
-import { List } from "../../Atoms";
+import { List } from "../../../atoms";
```

Comment:
Would be nice to be able to include from `/src` instead of going back up the tree. So `"atoms"` instead of `"../../../atoms"` Can you fix this in this PR or shall we make this an issue or a card in Trello?